Patches/Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_Great_Devourer.xml

```diff
@@ -20,13 +20,13 @@
 			<li Class="PatchOperationReplace">
 				<xpath>/Defs/ThingDef[defName="AA_GreatDevourer"]/statBases/ArmorRating_Blunt</xpath>
 				<value>
-					<ArmorRating_Blunt>1</ArmorRating_Blunt>
+					<ArmorRating_Blunt>8</ArmorRating_Blunt>
 				</value>
 			</li>
 			<li Class="PatchOperationReplace">
 				<xpath>/Defs/ThingDef[defName="AA_GreatDevourer"]/statBases/ArmorRating_Sharp</xpath>
 				<value>
-					<ArmorRating_Sharp>1</ArmorRating_Sharp>
+					<ArmorRating_Sharp>4</ArmorRating_Sharp>

```

Comment:
Nah. From the AA wiki: " Their whole name is in fact a pun, since they are not very resilient and are easily dispatched."